src/l500/l500-device.cpp

```diff
@@ -210,6 +210,9 @@ namespace librealsense
 
         if( _fw_version >= firmware_version( "1.5.0.0" ) )
         {
+            auto enable_max_usable_range = std::make_shared<max_usable_range_option>(this);
+            depth_sensor.register_option(RS2_OPTION_MAX_USABLE_RANGE, enable_max_usable_range);
```

Comment:
This option is really the enabler for max_usable_range and not the value itself.
Maybe confusing?
Call it `RS2_OPTION_ENABLE_MAX_USABLE_RANGE`?

Reply:
👍